src/math/random.js

```diff
@@ -19,11 +19,11 @@ var lcg = (function() {
   // Set to values from http://en.wikipedia.org/wiki/Numerical_Recipes
```

Comment:
could we update this comment to point to whatever library we're copying these values from?

Reply:
just forgot to check out comment. :smile: